  INFO root [0x00000ee4] [2018-08-06 09:41:22,925] - 
  INFO root [0x00000ee4] [2018-08-06 09:41:22,926] - -----------------------------------------------------------------------------------------


  INFO root [0x00000ee4] [2018-08-06 09:41:22,927] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00000ee4] [2018-08-06 09:41:22,927] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00000ee4] [2018-08-06 09:41:22,927] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00000ee4] [2018-08-06 09:41:22,948] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:5561 )
  INFO root [0x00000ee4] [2018-08-06 09:41:22,974] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:5561, SOMAXCONN=5.
  INFO root [0x00000ee4] [2018-08-06 09:41:22,974] - server-configs:
  INFO root [0x00000ee4] [2018-08-06 09:41:22,974] - 	gameUpdateHertz : 10
  INFO root [0x00000ee4] [2018-08-06 09:41:22,974] - 	defaultViewRadius : 80
  INFO root [0x00000ee4] [2018-08-06 09:41:22,975] - 	defaultViewHysteresisArea : 5
  INFO root [0x00000ee4] [2018-08-06 09:41:22,975] - 	entryScriptFile : kbemain
  INFO root [0x00000ee4] [2018-08-06 09:41:22,975] - 	internalAddr : 10.139.66.192:5561
  INFO root [0x00000ee4] [2018-08-06 09:41:22,975] - 	componentID : 8001
  INFO root [0x00000ee4] [2018-08-06 09:41:23,068] - Components::findLogger: found logger, addr:10.139.66.192:5503
  INFO root [0x00000ee4] [2018-08-06 09:41:23,069] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00000ee4] [2018-08-06 09:41:23,070] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000ee4] [2018-08-06 09:41:23,089] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 14084 ----
  INFO root [0x00000ee4] [2018-08-06 09:41:23,090] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00000ee4] [2018-08-06 09:41:23,090] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000ee4] [2018-08-06 09:41:23,091] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,258] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,260] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,262] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,266] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,267] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,267] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,268] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000ee4] [2018-08-06 09:41:23,347] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000ee4] [2018-08-06 09:41:23,417] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00000ee4] [2018-08-06 09:41:23,417] - Script::install(): KBExtra is successfully!
  INFO root [0x00000ee4] [2018-08-06 09:41:23,566] - TelnetServer server is running on port 50000
  INFO root [0x00000ee4] [2018-08-06 09:41:24,318] - ---- cellapp is running ----
 DEBUG root [0x00000ee4] [2018-08-06 09:41:24,319] - Components::process(): Request for the process of identity...
  WARN root [0x00000ee4] [2018-08-06 09:41:30,625] - Forwarding logs to logger[10.139.66.192:5503/0/0/0]...
 ERROR root [0x00000ee4] [2018-08-06 09:41:33,318] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5606/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:41:48,519] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5672/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:42:03,718] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5735/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:42:18,918] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5788/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:42:34,118] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5839/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:42:49,318] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5895/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:43:04,518] - PacketReader::processMessages: not found msgID=20035, msglen=49, from 127.0.0.1:5946/0/0/0.
 ERROR root [0x00000ee4] [2018-08-06 09:43:16,949] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000ee4] [2018-08-06 09:43:16,968] - The following logs sent to logger failed:
  WARN root [0x00000ee4] [2018-08-06 09:43:16,968] - ==> [2018-08-06 09:43:16 948] TCPPacketReceiver::processPendingEvents(10.139.66.192:5503): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000ee4] [2018-08-06 09:43:16,969] - ==> [2018-08-06 09:43:16 949] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000ee4] [2018-08-06 09:43:17,090] - TCPPacketReceiver::processPendingEvents(10.139.66.192:5521): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000ee4] [2018-08-06 09:43:17,092] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000ee4] [2018-08-06 09:43:17,101] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00000ee4] [2018-08-06 09:43:17,113] - Shutdowner::onShutdown: shutting down
  INFO root [0x00000ee4] [2018-08-06 09:43:17,114] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,115] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,115] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,133] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,133] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,134] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000ee4] [2018-08-06 09:43:17,147] - Script::uninstall(): is successfully!
 DEBUG root [0x00000ee4] [2018-08-06 09:43:17,148] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00000ee4] [2018-08-06 09:43:17,448] - ThreadPool::destroy(): waiting for thread(2)[0xea5b0,0xeb040,], try=1
  INFO root [0x000022d4] [2018-08-06 09:43:22,368] - 
  INFO root [0x000022d4] [2018-08-06 09:43:22,369] - -----------------------------------------------------------------------------------------


  INFO root [0x000022d4] [2018-08-06 09:43:22,370] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000022d4] [2018-08-06 09:43:22,371] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x000022d4] [2018-08-06 09:43:22,371] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:6026 )
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:6026, SOMAXCONN=5.
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - server-configs:
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - 	gameUpdateHertz : 10
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - 	defaultViewRadius : 80
  INFO root [0x000022d4] [2018-08-06 09:43:22,391] - 	defaultViewHysteresisArea : 5
  INFO root [0x000022d4] [2018-08-06 09:43:22,392] - 	entryScriptFile : kbemain
  INFO root [0x000022d4] [2018-08-06 09:43:22,392] - 	internalAddr : 10.139.66.192:6026
  INFO root [0x000022d4] [2018-08-06 09:43:22,392] - 	componentID : 8001
  INFO root [0x000022d4] [2018-08-06 09:43:22,420] - Components::findLogger: found logger, addr:10.139.66.192:6000
  INFO root [0x000022d4] [2018-08-06 09:43:22,420] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x000022d4] [2018-08-06 09:43:22,420] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000022d4] [2018-08-06 09:43:22,442] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 12832 ----
  INFO root [0x000022d4] [2018-08-06 09:43:22,443] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x000022d4] [2018-08-06 09:43:22,443] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000022d4] [2018-08-06 09:43:22,444] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,598] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,598] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,599] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,604] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,604] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,605] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,605] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000022d4] [2018-08-06 09:43:22,661] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000022d4] [2018-08-06 09:43:22,709] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x000022d4] [2018-08-06 09:43:22,709] - Script::install(): KBExtra is successfully!
  INFO root [0x000022d4] [2018-08-06 09:43:22,749] - TelnetServer server is running on port 50000
  INFO root [0x000022d4] [2018-08-06 09:43:23,346] - ---- cellapp is running ----
 DEBUG root [0x000022d4] [2018-08-06 09:43:23,347] - Components::process(): Request for the process of identity...
  WARN root [0x000022d4] [2018-08-06 09:43:29,624] - Forwarding logs to logger[10.139.66.192:6000/0/0/0]...
 ERROR root [0x000022d4] [2018-08-06 09:46:54,193] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000022d4] [2018-08-06 09:46:54,195] - The following logs sent to logger failed:
  WARN root [0x000022d4] [2018-08-06 09:46:54,195] - ==> [2018-08-06 09:46:54 194] TCPPacketReceiver::processPendingEvents(10.139.66.192:6000): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000022d4] [2018-08-06 09:46:54,196] - ==> [2018-08-06 09:46:54 194] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000022d4] [2018-08-06 09:46:54,358] - TCPPacketReceiver::processPendingEvents(10.139.66.192:6002): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000022d4] [2018-08-06 09:46:54,361] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000022d4] [2018-08-06 09:46:54,375] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x000022d4] [2018-08-06 09:46:54,379] - Shutdowner::onShutdown: shutting down
  INFO root [0x000022d4] [2018-08-06 09:46:54,379] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,380] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,381] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,407] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,408] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,408] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,408] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,409] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,409] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,409] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,409] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000022d4] [2018-08-06 09:46:54,435] - Script::uninstall(): is successfully!
 DEBUG root [0x000022d4] [2018-08-06 09:46:54,436] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000022d4] [2018-08-06 09:46:54,739] - ThreadPool::destroy(): waiting for thread(2)[0x2419dd0,0x241a248,], try=1
  INFO root [0x000037e8] [2018-08-06 09:46:57,702] - 
  INFO root [0x000037e8] [2018-08-06 09:46:57,703] - -----------------------------------------------------------------------------------------


  INFO root [0x000037e8] [2018-08-06 09:46:57,704] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000037e8] [2018-08-06 09:46:57,705] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x000037e8] [2018-08-06 09:46:57,705] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000037e8] [2018-08-06 09:46:57,725] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:6746 )
  INFO root [0x000037e8] [2018-08-06 09:46:57,726] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:6746, SOMAXCONN=5.
  INFO root [0x000037e8] [2018-08-06 09:46:57,726] - server-configs:
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	gameUpdateHertz : 10
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	defaultViewRadius : 80
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	defaultViewHysteresisArea : 5
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	entryScriptFile : kbemain
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	internalAddr : 10.139.66.192:6746
  INFO root [0x000037e8] [2018-08-06 09:46:57,727] - 	componentID : 8001
  INFO root [0x000037e8] [2018-08-06 09:46:57,914] - Components::findLogger: found logger, addr:10.139.66.192:6739
  INFO root [0x000037e8] [2018-08-06 09:46:57,914] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x000037e8] [2018-08-06 09:46:57,915] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000037e8] [2018-08-06 09:46:57,939] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 228 ----
  INFO root [0x000037e8] [2018-08-06 09:46:57,940] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x000037e8] [2018-08-06 09:46:57,942] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000037e8] [2018-08-06 09:46:57,942] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,094] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,095] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,096] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,101] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,101] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,101] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,102] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000037e8] [2018-08-06 09:46:58,166] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000037e8] [2018-08-06 09:46:58,242] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x000037e8] [2018-08-06 09:46:58,242] - Script::install(): KBExtra is successfully!
  INFO root [0x000037e8] [2018-08-06 09:46:58,353] - TelnetServer server is running on port 50000
  INFO root [0x000037e8] [2018-08-06 09:46:59,442] - ---- cellapp is running ----
 DEBUG root [0x000037e8] [2018-08-06 09:46:59,443] - Components::process(): Request for the process of identity...
  WARN root [0x000037e8] [2018-08-06 09:47:05,763] - Forwarding logs to logger[10.139.66.192:6739/0/0/0]...
 ERROR root [0x000037e8] [2018-08-06 10:00:34,569] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000037e8] [2018-08-06 10:00:34,571] - The following logs sent to logger failed:
  WARN root [0x000037e8] [2018-08-06 10:00:34,571] - ==> [2018-08-06 10:00:34 568] TCPPacketReceiver::processPendingEvents(10.139.66.192:6739): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000037e8] [2018-08-06 10:00:34,571] - ==> [2018-08-06 10:00:34 569] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000037e8] [2018-08-06 10:00:34,714] - TCPPacketReceiver::processPendingEvents(10.139.66.192:6740): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000037e8] [2018-08-06 10:00:34,718] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000037e8] [2018-08-06 10:00:34,721] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x000037e8] [2018-08-06 10:00:34,721] - Shutdowner::onShutdown: shutting down
  INFO root [0x000037e8] [2018-08-06 10:00:34,722] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,722] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,722] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,745] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,745] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,745] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,746] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,746] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,746] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,746] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,746] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000037e8] [2018-08-06 10:00:34,762] - Script::uninstall(): is successfully!
 DEBUG root [0x000037e8] [2018-08-06 10:00:34,762] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000037e8] [2018-08-06 10:00:35,064] - ThreadPool::destroy(): waiting for thread(2)[0x242a1f8,0x2429eb8,], try=1
  INFO root [0x00000dc8] [2018-08-06 10:00:37,704] - 
  INFO root [0x00000dc8] [2018-08-06 10:00:37,705] - -----------------------------------------------------------------------------------------


  INFO root [0x00000dc8] [2018-08-06 10:00:37,706] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00000dc8] [2018-08-06 10:00:37,706] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00000dc8] [2018-08-06 10:00:37,706] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00000dc8] [2018-08-06 10:00:37,731] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:9557 )
  INFO root [0x00000dc8] [2018-08-06 10:00:37,733] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:9557, SOMAXCONN=5.
  INFO root [0x00000dc8] [2018-08-06 10:00:37,733] - server-configs:
  INFO root [0x00000dc8] [2018-08-06 10:00:37,737] - 	gameUpdateHertz : 10
  INFO root [0x00000dc8] [2018-08-06 10:00:37,737] - 	defaultViewRadius : 80
  INFO root [0x00000dc8] [2018-08-06 10:00:37,737] - 	defaultViewHysteresisArea : 5
  INFO root [0x00000dc8] [2018-08-06 10:00:37,738] - 	entryScriptFile : kbemain
  INFO root [0x00000dc8] [2018-08-06 10:00:37,738] - 	internalAddr : 10.139.66.192:9557
  INFO root [0x00000dc8] [2018-08-06 10:00:37,738] - 	componentID : 8001
  INFO root [0x00000dc8] [2018-08-06 10:00:38,045] - Components::findLogger: found logger, addr:10.139.66.192:9550
  INFO root [0x00000dc8] [2018-08-06 10:00:38,045] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00000dc8] [2018-08-06 10:00:38,046] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000dc8] [2018-08-06 10:00:38,068] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 12696 ----
  INFO root [0x00000dc8] [2018-08-06 10:00:38,071] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00000dc8] [2018-08-06 10:00:38,073] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000dc8] [2018-08-06 10:00:38,074] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,235] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,236] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,237] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,241] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,242] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,242] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,243] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000dc8] [2018-08-06 10:00:38,315] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000dc8] [2018-08-06 10:00:38,366] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00000dc8] [2018-08-06 10:00:38,366] - Script::install(): KBExtra is successfully!
  INFO root [0x00000dc8] [2018-08-06 10:00:38,416] - TelnetServer server is running on port 50000
  INFO root [0x00000dc8] [2018-08-06 10:00:39,020] - ---- cellapp is running ----
 DEBUG root [0x00000dc8] [2018-08-06 10:00:39,022] - Components::process(): Request for the process of identity...
  WARN root [0x00000dc8] [2018-08-06 10:00:45,288] - Forwarding logs to logger[10.139.66.192:9550/0/0/0]...
 ERROR root [0x00000dc8] [2018-08-06 10:01:45,905] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000dc8] [2018-08-06 10:01:45,907] - The following logs sent to logger failed:
  WARN root [0x00000dc8] [2018-08-06 10:01:45,907] - ==> [2018-08-06 10:01:45 905] TCPPacketReceiver::processPendingEvents(10.139.66.192:9550): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000dc8] [2018-08-06 10:01:45,908] - ==> [2018-08-06 10:01:45 906] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000dc8] [2018-08-06 10:01:46,042] - TCPPacketReceiver::processPendingEvents(10.139.66.192:9551): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000dc8] [2018-08-06 10:01:46,044] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000dc8] [2018-08-06 10:01:46,071] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00000dc8] [2018-08-06 10:01:46,078] - Shutdowner::onShutdown: shutting down
  INFO root [0x00000dc8] [2018-08-06 10:01:46,079] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,079] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,079] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,097] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,098] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,099] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000dc8] [2018-08-06 10:01:46,116] - Script::uninstall(): is successfully!
 DEBUG root [0x00000dc8] [2018-08-06 10:01:46,117] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00000dc8] [2018-08-06 10:01:46,418] - ThreadPool::destroy(): waiting for thread(2)[0x29dadc0,0x29dae28,], try=1
  INFO root [0x000029d4] [2018-08-06 10:01:48,990] - 
  INFO root [0x000029d4] [2018-08-06 10:01:48,994] - -----------------------------------------------------------------------------------------


  INFO root [0x000029d4] [2018-08-06 10:01:48,995] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000029d4] [2018-08-06 10:01:48,995] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x000029d4] [2018-08-06 10:01:49,001] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000029d4] [2018-08-06 10:01:49,041] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:9832 )
  INFO root [0x000029d4] [2018-08-06 10:01:49,041] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:9832, SOMAXCONN=5.
  INFO root [0x000029d4] [2018-08-06 10:01:49,042] - server-configs:
  INFO root [0x000029d4] [2018-08-06 10:01:49,042] - 	gameUpdateHertz : 10
  INFO root [0x000029d4] [2018-08-06 10:01:49,042] - 	defaultViewRadius : 80
  INFO root [0x000029d4] [2018-08-06 10:01:49,042] - 	defaultViewHysteresisArea : 5
  INFO root [0x000029d4] [2018-08-06 10:01:49,042] - 	entryScriptFile : kbemain
  INFO root [0x000029d4] [2018-08-06 10:01:49,043] - 	internalAddr : 10.139.66.192:9832
  INFO root [0x000029d4] [2018-08-06 10:01:49,043] - 	componentID : 8001
  INFO root [0x000029d4] [2018-08-06 10:01:49,403] - Components::findLogger: found logger, addr:10.139.66.192:9825
  INFO root [0x000029d4] [2018-08-06 10:01:49,404] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x000029d4] [2018-08-06 10:01:49,404] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000029d4] [2018-08-06 10:01:49,428] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 11828 ----
  INFO root [0x000029d4] [2018-08-06 10:01:49,429] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x000029d4] [2018-08-06 10:01:49,430] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000029d4] [2018-08-06 10:01:49,433] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,595] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,595] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,596] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,601] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,602] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,602] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,603] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000029d4] [2018-08-06 10:01:49,675] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000029d4] [2018-08-06 10:01:49,747] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x000029d4] [2018-08-06 10:01:49,748] - Script::install(): KBExtra is successfully!
  INFO root [0x000029d4] [2018-08-06 10:01:49,837] - TelnetServer server is running on port 50000
  INFO root [0x000029d4] [2018-08-06 10:01:50,719] - ---- cellapp is running ----
 DEBUG root [0x000029d4] [2018-08-06 10:01:50,719] - Components::process(): Request for the process of identity...
  WARN root [0x000029d4] [2018-08-06 10:01:57,048] - Forwarding logs to logger[10.139.66.192:9825/0/0/0]...
 ERROR root [0x000029d4] [2018-08-06 10:03:15,460] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000029d4] [2018-08-06 10:03:15,464] - The following logs sent to logger failed:
  WARN root [0x000029d4] [2018-08-06 10:03:15,465] - ==> [2018-08-06 10:03:15 460] TCPPacketReceiver::processPendingEvents(10.139.66.192:9825): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000029d4] [2018-08-06 10:03:15,465] - ==> [2018-08-06 10:03:15 461] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000029d4] [2018-08-06 10:03:15,620] - TCPPacketReceiver::processPendingEvents(10.139.66.192:9826): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000029d4] [2018-08-06 10:03:15,622] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000029d4] [2018-08-06 10:03:15,635] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x000029d4] [2018-08-06 10:03:15,640] - Shutdowner::onShutdown: shutting down
  INFO root [0x000029d4] [2018-08-06 10:03:15,641] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,642] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,643] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,668] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,668] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,668] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,669] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,669] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,669] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,669] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,669] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000029d4] [2018-08-06 10:03:15,696] - Script::uninstall(): is successfully!
 DEBUG root [0x000029d4] [2018-08-06 10:03:15,696] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000029d4] [2018-08-06 10:03:15,997] - ThreadPool::destroy(): waiting for thread(2)[0x262ad58,0x262a7a8,], try=1
  INFO root [0x00001a68] [2018-08-06 10:03:18,517] - 
  INFO root [0x00001a68] [2018-08-06 10:03:18,519] - -----------------------------------------------------------------------------------------


  INFO root [0x00001a68] [2018-08-06 10:03:18,521] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001a68] [2018-08-06 10:03:18,521] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00001a68] [2018-08-06 10:03:18,522] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001a68] [2018-08-06 10:03:18,547] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:10178 )
  INFO root [0x00001a68] [2018-08-06 10:03:18,547] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:10178, SOMAXCONN=5.
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - server-configs:
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - 	gameUpdateHertz : 10
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - 	defaultViewRadius : 80
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - 	entryScriptFile : kbemain
  INFO root [0x00001a68] [2018-08-06 10:03:18,548] - 	internalAddr : 10.139.66.192:10178
  INFO root [0x00001a68] [2018-08-06 10:03:18,549] - 	componentID : 8001
  INFO root [0x00001a68] [2018-08-06 10:03:18,814] - Components::findLogger: found logger, addr:10.139.66.192:10172
  INFO root [0x00001a68] [2018-08-06 10:03:18,814] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00001a68] [2018-08-06 10:03:18,814] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001a68] [2018-08-06 10:03:18,838] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 10164 ----
  INFO root [0x00001a68] [2018-08-06 10:03:18,839] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00001a68] [2018-08-06 10:03:18,839] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001a68] [2018-08-06 10:03:18,840] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001a68] [2018-08-06 10:03:18,992] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a68] [2018-08-06 10:03:18,992] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001a68] [2018-08-06 10:03:18,993] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a68] [2018-08-06 10:03:18,999] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a68] [2018-08-06 10:03:19,000] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001a68] [2018-08-06 10:03:19,001] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a68] [2018-08-06 10:03:19,003] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a68] [2018-08-06 10:03:19,058] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a68] [2018-08-06 10:03:19,127] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00001a68] [2018-08-06 10:03:19,127] - Script::install(): KBExtra is successfully!
  INFO root [0x00001a68] [2018-08-06 10:03:19,192] - TelnetServer server is running on port 50000
  INFO root [0x00001a68] [2018-08-06 10:03:19,856] - ---- cellapp is running ----
 DEBUG root [0x00001a68] [2018-08-06 10:03:19,856] - Components::process(): Request for the process of identity...
  WARN root [0x00001a68] [2018-08-06 10:03:26,188] - Forwarding logs to logger[10.139.66.192:10172/0/0/0]...
 ERROR root [0x00001a68] [2018-08-06 10:05:24,954] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001a68] [2018-08-06 10:05:24,956] - The following logs sent to logger failed:
  WARN root [0x00001a68] [2018-08-06 10:05:24,957] - ==> [2018-08-06 10:05:24 955] TCPPacketReceiver::processPendingEvents(10.139.66.192:10172): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a68] [2018-08-06 10:05:24,957] - ==> [2018-08-06 10:05:24 956] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001a68] [2018-08-06 10:05:25,094] - TCPPacketReceiver::processPendingEvents(10.139.66.192:10173): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a68] [2018-08-06 10:05:25,095] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001a68] [2018-08-06 10:05:25,107] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001a68] [2018-08-06 10:05:25,107] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001a68] [2018-08-06 10:05:25,108] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,108] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,109] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,135] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,135] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,135] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,135] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,136] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,136] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,136] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,136] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a68] [2018-08-06 10:05:25,160] - Script::uninstall(): is successfully!
 DEBUG root [0x00001a68] [2018-08-06 10:05:25,160] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001a68] [2018-08-06 10:05:25,462] - ThreadPool::destroy(): waiting for thread(2)[0xa5a350,0xa5a420,], try=1
  INFO root [0x00003318] [2018-08-06 10:05:28,076] - 
  INFO root [0x00003318] [2018-08-06 10:05:28,084] - -----------------------------------------------------------------------------------------


  INFO root [0x00003318] [2018-08-06 10:05:28,087] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003318] [2018-08-06 10:05:28,088] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00003318] [2018-08-06 10:05:28,088] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003318] [2018-08-06 10:05:28,140] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:10656 )
  INFO root [0x00003318] [2018-08-06 10:05:28,141] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:10656, SOMAXCONN=5.
  INFO root [0x00003318] [2018-08-06 10:05:28,141] - server-configs:
  INFO root [0x00003318] [2018-08-06 10:05:28,141] - 	gameUpdateHertz : 10
  INFO root [0x00003318] [2018-08-06 10:05:28,141] - 	defaultViewRadius : 80
  INFO root [0x00003318] [2018-08-06 10:05:28,141] - 	defaultViewHysteresisArea : 5
  INFO root [0x00003318] [2018-08-06 10:05:28,142] - 	entryScriptFile : kbemain
  INFO root [0x00003318] [2018-08-06 10:05:28,142] - 	internalAddr : 10.139.66.192:10656
  INFO root [0x00003318] [2018-08-06 10:05:28,142] - 	componentID : 8001
  INFO root [0x00003318] [2018-08-06 10:05:29,658] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 4932 ----
  INFO root [0x00003318] [2018-08-06 10:05:29,659] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00003318] [2018-08-06 10:05:29,659] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003318] [2018-08-06 10:05:29,660] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00003318] [2018-08-06 10:05:29,838] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003318] [2018-08-06 10:05:29,839] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00003318] [2018-08-06 10:05:29,840] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00003318] [2018-08-06 10:05:29,845] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003318] [2018-08-06 10:05:29,846] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00003318] [2018-08-06 10:05:29,846] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003318] [2018-08-06 10:05:29,847] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003318] [2018-08-06 10:05:29,932] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00003318] [2018-08-06 10:05:29,995] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00003318] [2018-08-06 10:05:29,996] - Script::install(): KBExtra is successfully!
  INFO root [0x00003318] [2018-08-06 10:05:30,041] - TelnetServer server is running on port 50000
  INFO root [0x00003318] [2018-08-06 10:05:30,587] - ---- cellapp is running ----
 DEBUG root [0x00003318] [2018-08-06 10:05:30,587] - Components::process(): Request for the process of identity...
  WARN root [0x00003318] [2018-08-06 10:05:36,866] - EntityApp::checkTickPeriod: tick took 6278% (6.28 seconds)!
  INFO root [0x00003318] [2018-08-06 10:05:37,887] - Components::findComponents: find logger(1)...
  INFO root [0x00003318] [2018-08-06 10:05:37,978] - Components::findComponents: found logger, addr:10.139.66.192:10649
  INFO root [0x00003318] [2018-08-06 10:05:37,978] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00003318] [2018-08-06 10:05:37,978] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003318] [2018-08-06 10:05:37,990] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00003318] [2018-08-06 10:05:38,086] - Components::findComponents: found dbmgr, addr:10.139.66.192:10650
  INFO root [0x00003318] [2018-08-06 10:05:38,086] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00003318] [2018-08-06 10:05:38,086] - Components::addComponent[dbmgr], uid=10103, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003318] [2018-08-06 10:05:38,087] - Components::findComponents: found dbmgr, addr:10.139.66.192:10650
  WARN root [0x00003318] [2018-08-06 10:05:39,087] - EntityApp::checkTickPeriod: tick took 1300% (1.30 seconds)!
  WARN root [0x00003318] [2018-08-06 10:05:39,088] - Forwarding logs to logger[10.139.66.192:10649/0/0/0]...
 ERROR root [0x00003318] [2018-08-06 10:12:06,127] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00003318] [2018-08-06 10:12:06,130] - The following logs sent to logger failed:
  WARN root [0x00003318] [2018-08-06 10:12:06,130] - ==> [2018-08-06 10:12:06 127] TCPPacketReceiver::processPendingEvents(10.139.66.192:10649): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003318] [2018-08-06 10:12:06,130] - ==> [2018-08-06 10:12:06 128] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00003318] [2018-08-06 10:12:06,276] - TCPPacketReceiver::processPendingEvents(10.139.66.192:10650): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003318] [2018-08-06 10:12:06,286] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00003318] [2018-08-06 10:12:06,298] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00003318] [2018-08-06 10:12:06,306] - Shutdowner::onShutdown: shutting down
  INFO root [0x00003318] [2018-08-06 10:12:06,307] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00003318] [2018-08-06 10:12:06,308] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00003318] [2018-08-06 10:12:06,309] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00003318] [2018-08-06 10:12:06,336] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003318] [2018-08-06 10:12:06,336] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00003318] [2018-08-06 10:12:06,336] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00003318] [2018-08-06 10:12:06,337] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003318] [2018-08-06 10:12:06,337] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00003318] [2018-08-06 10:12:06,337] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003318] [2018-08-06 10:12:06,337] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003318] [2018-08-06 10:12:06,337] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00003318] [2018-08-06 10:12:06,360] - Script::uninstall(): is successfully!
 DEBUG root [0x00003318] [2018-08-06 10:12:06,360] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003318] [2018-08-06 10:12:06,662] - ThreadPool::destroy(): waiting for thread(2)[0x212e00,0x212648,], try=1
  INFO root [0x000018c8] [2018-08-06 10:12:10,317] - 
  INFO root [0x000018c8] [2018-08-06 10:12:10,317] - -----------------------------------------------------------------------------------------


  INFO root [0x000018c8] [2018-08-06 10:12:10,317] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000018c8] [2018-08-06 10:12:10,318] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x000018c8] [2018-08-06 10:12:10,318] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000018c8] [2018-08-06 10:12:10,340] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:12104 )
  INFO root [0x000018c8] [2018-08-06 10:12:10,340] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:12104, SOMAXCONN=5.
  INFO root [0x000018c8] [2018-08-06 10:12:10,340] - server-configs:
  INFO root [0x000018c8] [2018-08-06 10:12:10,340] - 	gameUpdateHertz : 10
  INFO root [0x000018c8] [2018-08-06 10:12:10,341] - 	defaultViewRadius : 80
  INFO root [0x000018c8] [2018-08-06 10:12:10,341] - 	defaultViewHysteresisArea : 5
  INFO root [0x000018c8] [2018-08-06 10:12:10,341] - 	entryScriptFile : kbemain
  INFO root [0x000018c8] [2018-08-06 10:12:10,341] - 	internalAddr : 10.139.66.192:12104
  INFO root [0x000018c8] [2018-08-06 10:12:10,341] - 	componentID : 8001
  INFO root [0x000018c8] [2018-08-06 10:12:10,372] - Components::findLogger: found logger, addr:10.139.66.192:12093
  INFO root [0x000018c8] [2018-08-06 10:12:10,372] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x000018c8] [2018-08-06 10:12:10,373] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000018c8] [2018-08-06 10:12:10,391] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 16:28:54 Apr 30 2018. AppID: 8001. UID: 10103. PID: 9264 ----
  INFO root [0x000018c8] [2018-08-06 10:12:10,391] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x000018c8] [2018-08-06 10:12:10,392] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000018c8] [2018-08-06 10:12:10,393] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,544] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,544] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,545] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,548] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,549] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,549] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,550] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018c8] [2018-08-06 10:12:10,634] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000018c8] [2018-08-06 10:12:10,723] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:35) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x000018c8] [2018-08-06 10:12:10,724] - Script::install(): KBExtra is successfully!
  INFO root [0x000018c8] [2018-08-06 10:12:10,799] - TelnetServer server is running on port 50000
  INFO root [0x000018c8] [2018-08-06 10:12:11,798] - ---- cellapp is running ----
 DEBUG root [0x000018c8] [2018-08-06 10:12:11,799] - Components::process(): Request for the process of identity...
  WARN root [0x000018c8] [2018-08-06 10:12:18,151] - Forwarding logs to logger[10.139.66.192:12093/0/0/0]...
 ERROR root [0x000018c8] [2018-08-06 10:21:58,929] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000018c8] [2018-08-06 10:21:58,930] - The following logs sent to logger failed:
  WARN root [0x000018c8] [2018-08-06 10:21:58,931] - ==> [2018-08-06 10:21:58 928] TCPPacketReceiver::processPendingEvents(10.139.66.192:12093): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018c8] [2018-08-06 10:21:58,931] - ==> [2018-08-06 10:21:58 930] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000018c8] [2018-08-06 10:21:59,031] - TCPPacketReceiver::processPendingEvents(10.139.66.192:12096): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018c8] [2018-08-06 10:21:59,033] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000018c8] [2018-08-06 10:21:59,039] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x000018c8] [2018-08-06 10:21:59,040] - Shutdowner::onShutdown: shutting down
  INFO root [0x000018c8] [2018-08-06 10:21:59,040] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,041] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,041] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,064] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,064] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,064] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,064] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,065] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,065] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,065] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,065] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000018c8] [2018-08-06 10:21:59,080] - Script::uninstall(): is successfully!
 DEBUG root [0x000018c8] [2018-08-06 10:21:59,081] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000018c8] [2018-08-06 10:21:59,382] - ThreadPool::destroy(): waiting for thread(2)[0xb5a260,0xb59f20,], try=1
  INFO root [0x00002538] [2018-08-06 10:30:18,321] - 
  INFO root [0x00002538] [2018-08-06 10:30:18,323] - -----------------------------------------------------------------------------------------


  INFO root [0x00002538] [2018-08-06 10:30:18,324] - Resmgr::initialize: KBE_ROOT=D:/kbe/kbengine-2.0.0-preview3/
  INFO root [0x00002538] [2018-08-06 10:30:18,325] - Resmgr::initialize: KBE_RES_PATH=D:/kbe/kbengine-2.0.0-preview3/kbe/res/;D:/kbe/kbengine-2.0.0-preview3/assets_Battle of Gods/;D:/kbe/kbengine-2.0.0-preview3/assets_Battle of Gods/scripts/;D:/kbe/kbengine-2.0.0-preview3/assets_Battle of Gods/res/
  INFO root [0x00002538] [2018-08-06 10:30:18,325] - Resmgr::initialize: KBE_BIN_PATH=D:/kbe/kbengine-2.0.0-preview3/kbe/bin/server/
  INFO root [0x00002538] [2018-08-06 10:30:18,333] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.89.70 ( 10.139.89.70:3446 )
  INFO root [0x00002538] [2018-08-06 10:30:18,333] - NetworkInterface::initialize(INTERNAL): address 10.139.89.70:3446, SOMAXCONN=5.
  INFO root [0x00002538] [2018-08-06 10:30:18,333] - server-configs:
  INFO root [0x00002538] [2018-08-06 10:30:18,333] - 	gameUpdateHertz : 10
  INFO root [0x00002538] [2018-08-06 10:30:18,333] - 	defaultViewRadius : 80
  INFO root [0x00002538] [2018-08-06 10:30:18,334] - 	defaultViewHysteresisArea : 5
  INFO root [0x00002538] [2018-08-06 10:30:18,334] - 	entryScriptFile : kbemain
  INFO root [0x00002538] [2018-08-06 10:30:18,334] - 	internalAddr : 10.139.89.70:3446
  INFO root [0x00002538] [2018-08-06 10:30:18,334] - 	componentID : 8001
  INFO root [0x00002538] [2018-08-06 10:30:18,354] - Components::findLogger: found logger, addr:10.139.89.70:3433
  INFO root [0x00002538] [2018-08-06 10:30:18,354] - Components::addComponent: reset orderLog, uid=21326!
  INFO root [0x00002538] [2018-08-06 10:30:18,355] - Components::addComponent[logger], uid=21326, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002538] [2018-08-06 10:30:18,366] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 84A43618D0FDA160D09DFF78C6735F0A. Config: Debug X86. Built: 23:21:46 May 26 2018. AppID: 8001. UID: 21326. PID: 6472 ----
  INFO root [0x00002538] [2018-08-06 10:30:18,366] - Server : Intel(R) Core(TM) i5-4210H CPU @ 2.90GHz with 2097151kB RAM
  INFO root [0x00002538] [2018-08-06 10:30:18,367] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002538] [2018-08-06 10:30:18,367] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002538] [2018-08-06 10:30:18,530] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002538] [2018-08-06 10:30:18,531] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002538] [2018-08-06 10:30:18,532] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002538] [2018-08-06 10:30:18,537] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002538] [2018-08-06 10:30:18,538] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002538] [2018-08-06 10:30:18,538] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002538] [2018-08-06 10:30:18,539] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002538] [2018-08-06 10:30:18,593] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002538] [2018-08-06 10:30:18,646] - Script::install(): is successfully, Python=(3.4.2 (default, May 26 2018, 23:19:59) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002538] [2018-08-06 10:30:18,646] - Script::install(): KBExtra is successfully!
  INFO root [0x00002538] [2018-08-06 10:30:18,692] - TelnetServer server is running on port 50000
  INFO root [0x00002538] [2018-08-06 10:30:19,125] - ---- cellapp is running ----
 DEBUG root [0x00002538] [2018-08-06 10:30:19,126] - Components::process(): Request for the process of identity...
  WARN root [0x00002538] [2018-08-06 10:30:25,305] - Forwarding logs to logger[10.139.89.70:3433/0/0/0]...
 ERROR root [0x00002538] [2018-08-06 10:31:19,319] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002538] [2018-08-06 10:31:19,332] - The following logs sent to logger failed:
  WARN root [0x00002538] [2018-08-06 10:31:19,333] - ==> [2018-08-06 10:31:19 316] TCPPacketReceiver::processPendingEvents(10.139.89.70:3433): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002538] [2018-08-06 10:31:19,333] - ==> [2018-08-06 10:31:19 319] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002538] [2018-08-06 10:31:19,495] - TCPPacketReceiver::processPendingEvents(10.139.89.70:3435): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002538] [2018-08-06 10:31:19,502] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002538] [2018-08-06 10:31:19,542] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002538] [2018-08-06 10:31:19,543] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002538] [2018-08-06 10:31:19,555] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002538] [2018-08-06 10:31:19,595] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002538] [2018-08-06 10:31:19,599] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002538] [2018-08-06 10:31:19,708] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002538] [2018-08-06 10:31:19,709] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002538] [2018-08-06 10:31:19,710] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002538] [2018-08-06 10:31:19,747] - Script::uninstall(): is successfully!
 DEBUG root [0x00002538] [2018-08-06 10:31:19,751] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002538] [2018-08-06 10:31:20,059] - ThreadPool::destroy(): waiting for thread(2)[0x1b9aa98,0x1b9a4e8,], try=1
